tests/GoutteSimpleTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
```

Comment:
Why you've added a new test case? Doesn't the existing test suite work with a new Goutte version?